packages/asc-ui/src/components/Button/ButtonStyle.ts

```diff
@@ -241,19 +256,6 @@ const ButtonStyle = styled.button<Props>`
         opacity: 1;
       }
     `}
-  ${({ variant }) =>
```

Comment:
Why move this styling up? Looked like it's only applied to the `:disabled` state of the button